types/mongration/index.d.ts

```diff
@@ -0,0 +1,35 @@
+// Type definitions for mongration 1.0
+// Project: https://github.com/awapps/mongration#readme
+// Definitions by: Anton Lobashev <https://github.com/soulthreads>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+import { Db } from 'mongodb';
+
+export interface DbConfig {
+    hosts?: string;
+    db?: string;
+    user?: string;
+    password?: string;
+    mongoUri?: string;
+    migrationCollection: string;
+    replicaSet?: string;
+}
+
+export interface MigrationResponse {
+    id: string;
+    status: string;
+}
+
+export class Migration {
+    constructor(dbConfig: DbConfig);
+    add: (paths: string | string[]) => void;
+    addAllFromPath: (path: string) => void;
```

Comment:
Do not see this any where on readme. https://github.com/awapps/mongration#adding-steps-to-migration is this intended to be exposed?

Reply:
Yeah, they have it in their samples: https://github.com/awapps/mongration/blob/master/samples/03-folder-scan/migration.js#L14